x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/results/Bucket.java

```diff
@@ -167,6 +184,10 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().before(Version.V_5_5_0)) {
             out.writeGenericValue(Collections.emptyMap());
         }
+        // bwc for perPartitionNormalization
+        if (out.getVersion().before(Version.V_6_5_0)) {
+            out.writeGenericValue(Collections.emptyList());
```

Comment:
I was expecting this to be `out.writeList(Collections.emptyList());`. Did you try that out?

Reply:
I'll make the change now - you're right, writeList is the better option here